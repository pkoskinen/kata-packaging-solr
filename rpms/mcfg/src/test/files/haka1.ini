[{ "spname" : "%%SPNAME%%" } , { "more_stuff" : "%%SOME_OTHER%%" }]