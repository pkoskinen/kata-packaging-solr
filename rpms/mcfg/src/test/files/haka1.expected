[{ "spname" : "kata-test1.csc.fi" } , { "more_stuff" : "foo" }]